components/canvas/canvas_paint_task.rs

```diff
@@ -603,6 +614,51 @@ impl FillOrStrokeStyle {
     }
 }
 
+#[repr(i32)]
+#[derive(Clone, Copy, PartialEq)]
```

Comment:
There are some elements like this, which are copies of their azure counterparts. We probably don't want to include azure in the dom, but I wonder if there's a better way to do this?